src/libAtomVM/term.h

```diff
@@ -2396,16 +2396,18 @@ static inline term term_from_float(avm_float_t f, Heap *heap)
     term *boxed_value = memory_heap_alloc(heap, FLOAT_SIZE);
     boxed_value[0] = ((FLOAT_SIZE - 1) << 6) | TERM_BOXED_FLOAT;
 
-    float_term_t *boxed_float = (float_term_t *) (boxed_value + 1);
-    boxed_float->f = f;
-
+    // For wasm32, alignof(avm_float_t) = 8 and alignof(term) = 4
```

Comment:
This comment is somehow slightly misleading, this is not just an issue for wasm32.
It should be rephrased as:
// alignof(avm_float_t) might be != alignof(term)